setup.cfg

```diff
@@ -40,6 +40,7 @@ install_requires =
     ftfy>=4.4.3,<5.0.0
     invenio-administration>=1.8.0,<2.0.0
     invenio-communities>=7.0.0,<8.0.0
+    invenio-records-resources>=4.17.1,<5.0.0  # TODO: change to version that introduces the fix
```

Comment:
I am just getting tired of going through the invenio-drafts-resources middleman to get at invenio-records-resources. invenio-rdm-records uses invenio-records-resources directly, so having a direct dependency on it is fair. I don't think invenio-drafts-resources is one of the official "intermediary" packages.